tests/benchmarks/shared.py

```diff
@@ -231,3 +231,11 @@ class Foo:
     FutureDate,
     PastDatetime,
 ]
+
+
+class DeferredModel(BaseModel):
+    model_config = {'defer_build': True}
+
+
+def rebuild_model(model: Type[BaseModel]) -> None:
+    model.model_rebuild(force=True, _types_namespace={})
```

Comment:
This is fine bc it's working now, but if we do more complex model rebuilding, we might need to remove this types namespace specification.